packages/webapi/models/requests.go

```diff
@@ -15,7 +15,8 @@ type ContractCallViewRequest struct {
 }
 
 type EstimateGasRequestOnledger struct {
-	Output string `json:"outputBytes" swagger:"desc(Serialized Output (Hex)),required"`
+	DryRunRes string `json:"dryRunRes" swagger:"desc(The result of the DryRun encoded in BCS format and represented in hexadecimal),required"`
```

Comment:
Asked the user to call DryRun, because not sure if it is ok to access L1Client in `packages/webapi/controllers/chain/estimategas.go`